src/EnergyPlus/UnitarySystem.cc

```diff
@@ -12198,6 +12206,17 @@ namespace UnitarySystems {
                 } else {
                     this->m_SuppHeatPartLoadFrac = 1.0;
                 }
+            } else {
+                mdot = 0.0;
+                state.dataLoopNodes->Node(this->m_SuppCoilFluidInletNode).MassFlowRate = mdot;
+                //     simulate water coil to find operating capacity
+                WaterCoils::SimulateWaterCoilComponents(state,
+                                                        this->m_SuppHeatCoilName,
+                                                        FirstHVACIteration,
+                                                        this->m_SuppHeatCoilIndex,
+                                                        QActual,
+                                                        this->m_FanOpMode,
+                                                        this->m_SuppHeatPartLoadFrac);
```

Comment:
Not sure how this coil wound up not calling the coil model when there was no supplemental heating load. I also wonder if this is the only coil model doing that?

Reply:
I see, lines 12181 was changed to 12182.

            // if (present(SuppCoilLoad)) {
            if (SuppHeatCoilLoad > 0.0) {